software/qualimap/bamqc/main.nf

```diff
@@ -0,0 +1,76 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process QUALIMAP_BAMQC {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    conda (params.enable_conda ? "bioconda::qualimap=2.2.2d" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/qualimap:2.2.2d--1"
+    } else {
+        container "quay.io/biocontainers/qualimap:2.2.2d--1"
+    }
+
+    input:
+    tuple val(meta), path(bam)
+    path regions
+
+    output:
+    tuple val(meta), path("${prefix}"), emit: results
+    path  "*.version.txt"             , emit: version
+
+    script:
+    def software   = getSoftwareName(task.process)
+    prefix         = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+
+    def memory = 3
+    if (!task.memory) {
+        log.info '[Qualimap BamQC] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
+    } else {
+        memory = task.memory.toGiga()
+    }
+    def threads    = task.cpus
```

Comment:
I think we should remove this. The [`qualimap/rnaseq`](https://github.com/nf-core/modules/blob/0a411302767d133d6af1bf5c82cc9b25db57a3b6/software/qualimap/rnaseq/main.nf#L42-L53) module I added has been working fine without it so far. I would assume it should be ok.